lib/afip_bill/generator.rb

```diff
@@ -68,5 +69,9 @@ def code_hash
     def template
       ERB.new(File.read(bill_path)).result(binding)
     end
+
+    def should_hide_service_dates
```

Comment:
@tcordeu A small code style change: `hide_service_dates?` instead of `should_hide_service_dates`